torch_geometric/graphgym/logger.py

```diff
@@ -10,6 +13,16 @@
 from torch_geometric.graphgym.utils.device import get_current_gpu_usage
 from torch_geometric.graphgym.utils.io import dict_to_json, dict_to_tb
 
+try:
```

Comment:
I am wondering if there is some better way to do this besides having this `try`/`catch`-block in every file. Maybe we do this in a separate file once (e.g., `import.py`) and import from there?

Reply:
yeah, make sense